src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (invertedRobot.hasOwnProperty(robot[key])) {
+      return null;
+    }
+
+    invertedRobot[String(robot[key])] = key;
+  }
+
+  return invertedRobot;
 }
 
 module.exports = inverseRobot;
```

Comment:
It's good practice to include a newline at the end of the file. Some tools and systems expect or require it for proper processing.